src/plugins/expressions/common/ast/format.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ExpressionAstExpression, ExpressionAstArgument } from './types';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { toExpression } = require('@kbn/interpreter/common');
+
+export function format(
+  ast: ExpressionAstExpression | ExpressionAstArgument,
+  type: 'expression' | 'argument'
+): string {
+  return toExpression(ast, type);
+}
+
+export function formatExpression(ast: ExpressionAstExpression): string {
+  return format(ast, 'expression');
+}
```

Comment:
This is just a subjective naming nit, but personally `fromExpression` and `toExpression` made much more sense to me in terms of describing what they did.

`parseExpression` makes sense, but for whatever reason `formatExpression` to me feels unclear in its intent...

If I were looking for a function that could take my AST and turn it into an expression string, `formatExpression` would not be my first guess; I'd be looking for something like `toExpressionString` `convertToExpressionString`, `astToString` etc.

Reply:
I've used "parse" as a name for parsing from string to AST, and my intent for "format" was to signal that it is a reverse of "parse". But I agree, "format" is not the best choice. I'm going to leave this as is for now to not modify this PR even more, but in general I'm happy with all above suggestions we can use later to rename this function. Also, I've just looked at parsers and an often used term for this is "generate" or "codegen", maybe we could use `generateExpression`. Another term I've seen is "print", not sure if we want to use it though.

Reply:
Mentioned it here: https://github.com/elastic/kibana/issues/57340